app/controllers/proposals_controller.rb

```diff
@@ -216,6 +216,6 @@ def event_params
   end
 
   def user_params
-    params.require(:user).permit(:email, :password, :password_confirmation, :username)
+    params.require(:user).permit(:email, :password, :password_confirmation, :username, :is_admin)
```

Comment:
Hmm, why is this param needed on the proposals page? This could in theory allow someone to create a user as an admin, since these params (I don't think...) aren't filtered before creating / updating users.

Reply:
yeah this is my mistake. fixed.